downstreamadapter/dispatchermanager/dispatcher_manager.go

```diff
@@ -87,6 +87,9 @@ type DispatcherManager struct {
 	dispatcherMap *DispatcherMap[*dispatcher.EventDispatcher]
 	// redoDispatcherMap restore all the redo dispatchers in the DispatcherManager, including redo table trigger event dispatcher
 	redoDispatcherMap *DispatcherMap[*dispatcher.RedoDispatcher]
+	// currentOperatorMap stores the current operators for each dispatcher
+	currentOperatorMap     sync.Map // map[span.String()]*dispatcher.CurrentOperator
```

Comment:
span.String() means dispatcherID?

Reply:
I believe that currently only the span ID can be used as a unique identifier. The dispatcher ID may change, and the operator essentially wants to schedule the span, so it is necessary to ensure that there is only one span globally.